src/Wallabag/CoreBundle/Entity/Comment.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Wallabag\CoreBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * Tag.
+ *
+ * @ORM\Table
+ * @ORM\Entity(repositoryClass="Wallabag\CoreBundle\Repository\CommentRepository")
+ */
+class Comment
+{
+	/**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="content", type="text")
+     */
+    private $content;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Entry", inversedBy="comments")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $entry;
+
+	/**
+	 * @var datetime $date
+	 *
+	 * @ORM\Column(name="date", type="datetime")
+	 */
+	private $date;
```

Comment:
I was looking yesterday for such a tool and found PHP_CodeSniffer but wasn't totally satisfied. Thanks @j0k3r, I will have a look too :)